.gitlab-ci.yml

```diff
@@ -264,10 +257,62 @@ build-nightly:
 
 #### stage:                        publish
 
+# check that images can be built
+.build-image:                      &build-image
+  <<:                              *kubernetes-build
+  image:                           $BUILDAH_IMAGE
+  <<:                              *test-refs
+  variables:                       &build-image-variables
+    GIT_STRATEGY:                  none
+    DOCKERFILE:                    ci.Dockerfile
+  needs:
+    - job:                         build
+      artifacts:                   true
+  script:
+    # trim "-build-docker" from job name
+    - export DOCKER_IMAGE_NAME="${CI_JOB_NAME::-13}"
+    - expoty BRIDGES_PROJECT="${CI_JOB_NAME::-13}"
```

Comment:
```suggestion
    - export BRIDGES_PROJECT="${CI_JOB_NAME::-13}"
```

Reply:
fixed in https://github.com/paritytech/parity-bridges-common/pull/2058/commits/b6537a1bcf8180412df33bc3e9aa91ead6e4f7d9